README.md

```diff
@@ -91,3 +91,17 @@ vagrant up --provision
 ```
 * Go to localhost:1111 in your browser
 
+
+## Running Docker
+
+* Install Docker Desktop https://docs.docker.com/docker-for-windows/install/
+
+* Build docker image
+```
+docker build --target development --tag todo-app:dev . 
+```
+* Run docker image
+```
+docker run todo-app:dev
+```
```

Comment:
You are missing three important options - publishing a port, specifying the .env file, adding a bind mount.
But you have those correctly set in your docker-compose file.
````suggestion
* To run the Docker container:
```
docker-compose up --build
```
````

Reply:
It would be good to also document how to build/run the production version.